### PR TITLE
Can I please have a larger volume?

### PR DESCRIPTION
**Pull request by sgosline**
I hope this is the right way.

### CODE REVIEW

File: config/prod/nf-osi-processing.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+parameters:
+  # The Sage deparment (Platform, CompOnc, SysBio, Governance, etc..)
+  Department: "CompOnc"
+  # The Sage project (Infrastructure, amp-ad, ntap, dream, etc..)
+  Project: "NF OSI"
+  # The resource owner
+  OwnerEmail: "sara.gosline@sagebase.org"
+  # (Optional) EC2 instance type, default is "t2.nano" (other available types https://aws.amazon.com/ec2/instance-types/)
+  InstanceType: "t2.2xlarge"
+  VolumeSize: 500
```
Comment (thomasyu888) [inline config/prod/nf-osi-processing.yaml]: Needs to be a string. "500"

Comment (zaro0508) [inline config/prod/nf-osi-processing.yaml]: you need to put quotes around numbers like so `VolumeSize: "500"`

File: config/prod/nf-osi-processing.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  JcConnectKey: !ssm /infra/JcConnectKey
+  JcServiceApiKey: !ssm /infra/JcServiceApiKey
+  JcSystemsGroupId: !ssm /infra/JcSystemsGroupId
+hooks:
+  after_create:
+    - !ec2_notify {{stack_group_config.aws_account_name}} {{stack_group_config.aws_account_email}}
```
Comment (zaro0508) [inline config/prod/nf-osi-processing.yaml]: we added a syntax check for this.  it needs to have quotes, like so..

```
!ec2_notify "{{stack_group_config.aws_account_name}} {{stack_group_config.aws_account_email}}"
```

**Comment by sgosline** [comment]
Cant I just resize the volume? I already installed a bunch of stuff :-/

**Pull request closed without merging**